utils/docker/Dockerfile.leap.15

```diff
@@ -209,11 +209,13 @@ USER daos_server:daos_server
 # select compiler to use
 ARG COMPILER=gcc
 ARG JOBS=$DEPS_JOBS
+ARG DAOS_BUILD_TYPE=release
 ARG DAOS_BUILD=$DAOS_DEPS_BUILD
 
 # Build DAOS
 RUN [ "$DAOS_BUILD" != "yes" ] || \
-    { scons --jobs $JOBS install PREFIX=/opt/daos COMPILER=$COMPILER && \
+    { scons --jobs $JOBS install PREFIX=/opt/daos \
+	COMPILER=$COMPILER BUILD_TYPE=$DAOS_BUILD_TYPE TARGET_TYPE=release && \
```

Comment:
If you for any reason need to rebase this patch, if you could add TARGET_TYPE as an argument with the default value being "release", it can address a customer request to be able to build prerequisites with debug using the docker script.  If not, I will wait for this to land

Reply:
Done, there should be a DAOS_TARGET_TYPE option now as well.